src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -80,13 +80,10 @@ private void WriteClientMethodBody(CodeWriter writer, LowLevelClientMethod clien
                     writer.Append($"{parameter.Name:I}, ");
                 }
                 writer.RemoveTrailingComma();
-                writer.Append($");");
+                writer.Line($");");
```

Comment:
NIT: thanks for using `.Line()` here instead.  Can we remove the extra `writer.Line()` call that follows so we don't have a blank line in the generated output?